### ISSUE TITLE
Photo Editor App

### ISSUE THREAD

**Issue by abhisheks008** [World-of-ML/Play-With-Python#77]
### Play With Python Repository (Proposing new issue)
:red_circle: **Project Title** : Photo Editor App</br>
:red_circle: **Aim** : Create a photo editor app using Python programming language. </br>

**********************************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by mathew4STAR** [comment]
Hey, Could you explain what all features you expect for this?

**Comment by abhisheks008** [comment]
1. Crop feature
2. Brightness, contrast, saturation adjustments.
3. Few filters.
4. Rotate the image feature.
5. Add a text feature.

Initially these are required.
@mathew4STAR

**Comment by mathew4STAR** [comment]
Can you please assign this issue to me, I would like to attempt it.

**Comment by bcsamrudh** [comment]
@abhisheks008 .Can we take up these issues?

**Comment by abhisheks008** [comment]
Yeah sure, all those existing issues will be considered for DWOC.

**Comment by bcsamrudh** [comment]
Full name : BC SAMRUDH
GitHub Profile Link : https://github.com/bcsamrudh
Email ID : [samrudhbc@gmail.com](mailto:samrudhbc@gmail.com)
Participant ID (if applicable):
Approach for this Project : I would use OpenCV and PIL(pillow) libraries.
What is your participant role?: DWOC23

**Comment by abhisheks008** [comment]
Issue assigned to you @bcsamrudh
